app/javascript/components/student/editor/ExercismMonacoEditor.tsx

```diff
@@ -44,18 +44,17 @@ export function ExercismMonacoEditor({
   value: string | null | undefined
   theme: string
 }) {
-  const handleEditorDidMount = (
-    editor: monacoEditor.editor.IStandaloneCodeEditor
-  ) => {
-    editor.addAction({
-      id: 'runTests',
-      label: 'Run tests',
-      keybindings: [monacoEditor.KeyCode.F2],
-      run: onRunTests,
-    })
+  const startLanguageClient = useCallback(() => {
+    const languageServerUrl = document.querySelector<HTMLMetaElement>(
+      'meta[name=language-server-url]'
```

Comment:
```suggestion
      'meta[name="language-server-url"]'
```

Both are correct, but quotes make it harder to break when changed.